pymc/model_graph.py

```diff
@@ -234,20 +242,100 @@ def make_graph(self, var_names: Optional[Iterable[VarName]] = None, formatting:
 
         return graph
 
+    def make_networkx(self, var_names: Optional[Iterable[VarName]] = None, formatting: str = "plain"):
+        """Make networkx Digraph of PyMC model
+        Returns
+        -------
+        networkx.Digraph
+        """
+        try:
+            import networkx
+        except ImportError:
+            raise ImportError(
+                "This function requires the python library networkx, along with binaries. "
+                "The easiest way to install all of this is by running\n\n"
+                "\tconda install networkx"
+            )
+        graphnetwork = networkx.DiGraph(name=self.model.name)
+        for plate_label, all_var_names in self.get_plates(var_names).items():
+            if plate_label:
+                # # must be preceded by 'cluster' to get a box around it
+                subgraphnetwork = networkx.DiGraph(name="cluster" + plate_label,label=plate_label)
+                for var_name in all_var_names:
+                    self._make_node(var_name, subgraphnetwork,  nx=True, cluster="cluster" + plate_label, formatting=formatting,)    
+                for sgn in subgraphnetwork.nodes:
+                    networkx.set_node_attributes(subgraphnetwork,{sgn:{'labeljust':"r", 'labelloc':"b", 'style':"rounded"}})
+                node_data = {e[0]:e[1] for e in graphnetwork.nodes(data=True) & subgraphnetwork.nodes(data=True)}
+                
+                graphnetwork = networkx.compose( graphnetwork, subgraphnetwork )  
+                networkx.set_node_attributes(graphnetwork,node_data)  
+                graphnetwork.graph['name']=self.model.name 
+            else:
+                for var_name in all_var_names:
+                    self._make_node(var_name, graphnetwork, nx=True, formatting=formatting)
+
+        for child, parents in self.make_compute_graph(var_names=var_names).items():
+            # parents is a set of rv names that preceed child rv nodes
+            for parent in parents:
+                graphnetwork.add_edge(parent.replace(":", "&"), child.replace(":", "&"))
+        return graphnetwork
+
+
+def model_to_networkx(
+    model=None, *, var_names: Optional[Iterable[VarName]] = None, formatting: str = "plain"
+):
+    """Produce a networkx Digraph from a PyMC model.
+    Requires networkx, which may be installed most easily with
```

Comment:
```suggestion
    """Produce a networkx Digraph from a PyMC model.
    
    Requires networkx, which may be installed most easily with
```

There must be a new line between the short summary and the extended description. Docstring formatting should follow [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html), I recommend you go over https://pymc-data-umbrella.xyz/en/latest/sprint/tutorials/docstring_tutorial.html#edit-the-docstring.

Moreover, the lack of empty lines between sections is breaking the rendering of the docstring. Docstrings are written in rst which is not markdown but it has many similarities like for example, ignoring breaklines and adding breaklines only when an empty line is present. 

* model_to_graphviz docstring:
  - With this PR: https://pymcio--6046.org.readthedocs.build/projects/docs/en/6046/api/generated/pymc.model_to_graphviz.html
  - on latest https://www.pymc.io/projects/docs/en/latest/api/generated/pymc.model_to_graphviz.html

The same way you need to add the spaces here, you need to _keep_ them on model_to_graphviz, your PR is currently deleting all the empty lines in that docstring (which is the one I linked to show the rendering issues). 

Also, you need to add this new function to the docs, you should list it in https://github.com/pymc-devs/pymc/blob/main/docs/source/api/model.rst right under model_to_graphviz

Reply:
Ok I've done both of those.
Thanks for highlighting